### PR TITLE
Exercise 1

### PR DESCRIPTION
**Pull request by vaibhav1sh**
New pull

### CODE REVIEW

File: todo_app/app.py [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
 def index():
-    return 'Hello World!'
+    todos = session_items.get_items()
+    return render_template('index.html',todos=todos)
 
+@app.route('/add-item', methods = ["POST"])
+def add_item():
+    title = request.form['title_of_todo']
+    if (title != ""):
+        session_items.add_item(title)
```
Comment (Jonesey13) [inline todo_app/app.py:19]: Maybe worth adding a message here when the title is empty

File: todo_app/templates/index.html [hunk 2]
```diff
@@ -12,7 +12,25 @@ <h1 class="display-4">To-Do App</h1>
...
     </div>
   </div>
+
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add New Items</h2>
+      <form action="/add-item" method="POST">
+        <ul class="list-group mb-4">
+          <li><label for ="title">Title : </label></li>
+          <input type="text" id="title" name="title_of_todo">
```
Comment (Jonesey13) [inline todo_app/templates/index.html:28]: You can add some client side validation using the `required` attribute:
```suggestion
          <input type="text" id="title" name="title_of_todo" required>
```

**Review by Jonesey13 — APPROVED** [review]
Looks all good to me 👍 

Feel free to merge this PR in when ready.

**Comment by vaibhav1sh** [comment]
Merged

**Pull request merged**